.circleci/workflows.yml

```diff
@@ -1010,12 +1492,12 @@ workflows:
 
   sonarcloud_analysis_back:
```

Comment:
Can't see conditions to avoid execution on Release, DockerImage build & RPM build

Reply:
OK, because it is linked to another issue :)